app/models/property.rb

```diff
@@ -65,6 +65,19 @@ def address_complete?
   end
 
   def property_owners_complete?
-    property_owners.all?(&:complete?)
+    return true if has_other_owners == YesNoAnswer::NO.to_s || has_other_owners.nil?
```

Comment:
Is this guard needed? feels like line 70 is sufficient?

Reply:
Just trying to save adding query when not required :)